azafea_metrics_proxy/config/__init__.py

```diff
@@ -76,6 +76,7 @@ class Redis(_Base):
     host: str = 'localhost'
     port: int = 6379
     password: str = DEFAULT_PASSWORD
+    ssl: bool = False
```

Comment:
Sadly this has broken the tests in a boring fashion.

Reply:
I think... although running the tests locally shows a pile of failed integration tests.

Reply:
To run the integration tests locally you probably need to manually spin up an appropriate Redis...